src/ape/utils/abi.py

```diff
@@ -132,6 +136,9 @@ def _create_struct(self, out_abi: ABIType, out_value) -> Optional[Any]:
             name = out_abi.name or self.default_name
 
         components = self._parse_components(out_abi.components, out_value[0])
+        if not components:
+            return None  # Empty struct
```

Comment:
When return `empty(MyStruct)` in Vyper, it gives me a components like this: 
```
['0x0000000000000000000000000000000000000000', HexBytes('0x0000000000000000000000000000000000000000000000000000000000000000')]
```

Is `empty(MyStruct)` not what you mean by `# Empty struct` here?

Reply:
Were you able to hit this return statement in your example?

Reply:
`empty(MyStruct)` means "a `MyStruct` with all default (empty) values", so it actually does produce a struct (which is perhaps confusing)

Reply:
An "empty struct" is a struct w/o any components, e.g. `struct MyStruct: pass`, which I don't think is valid in Vyper (and maybe Solidity)

Reply:
Perhaps this should raise instead

Reply:
It is needed for empty tuples actually, but I adjusted it a little.